cookbooks/bcpc-hadoop/templates/default/flume_flume-conf.erb

```diff
@@ -1,5 +1,6 @@
 <%= @agent_name %>.channels = memoryChannel
 <%= @agent_name %>.channels.memoryChannel.type = memory
+<%= @agent_name %>.channels.memoryChannel.capacity = 10000
```

Comment:
How do we decide what this value will be (e.g.: 10000 in this case)? random/recommended/empirical?